pkg/cmd/cli/bug/bug.go

```diff
@@ -0,0 +1,203 @@
+/*
+Copyright 2018 the Heptio Ark contributors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package bug
+
+import (
+	"bytes"
+	"errors"
+	"fmt"
+	"net/url"
+	"os"
+	"os/exec"
+	"runtime"
+	"strings"
+	"text/template"
+	"time"
+
+	"github.com/heptio/ark/pkg/buildinfo"
+	"github.com/heptio/ark/pkg/cmd"
+	"github.com/spf13/cobra"
+)
+
+const (
+	// kubectlTimeout is how long we wait in seconds for `kubectl version`
+	// before killing the process
+	kubectlTimeout = 5 * time.Second
+	issueURL       = "https://github.com/heptio/ark/issues/new"
+	// IssueTemplate is used to generate .github/ISSUE_TEMPLATE/bug_report.md
+	// as well as the initial text that's place in a new Github issue as
+	// the result of running `ark bug`.
+	IssueTemplate = `---
```

Comment:
@jhamilton1 This will be the location for the bug template, if we merge this. What it means is that the Go file will have to be updated, then we run the `update-generated-issue-template.sh` script in this repo to get the file that GitHub uses. That enables us to not repeat the template in 2 files.

Is this approach ok with you?

Reply:
Works for us @nrb

Reply:
Awesome, thanks!